invoices/helpers_test.go

```diff
@@ -0,0 +1,209 @@
+package invoices
```

Comment:
A separate commit that does a pure move makes it easier to review the addition. But no need to change now.

Reply:
Can separate it if you really want to...